uwrt_mars_rover_utils/include/uwrt_mars_rover_utils/uwrt_can.h

```diff
@@ -17,20 +17,22 @@ namespace uwrt_mars_rover_utils {
 class UWRTCANWrapper {
  public:
   explicit UWRTCANWrapper() = default;
-  explicit UWRTCANWrapper(
-      std::string name, std::string interface_name, bool rcv_big_endian,
-      int thread_sleep_millis = 10);  // NOLINT(cppcoreguidelines-avoid-magic-numbers, readability-magic-numbers)
-  explicit UWRTCANWrapper(const UWRTCANWrapper& to_copy) = delete;
-  // NOLINTNEXTLINE(performance-noexcept-move-constructor, bugprone-exception-escape)
-  explicit UWRTCANWrapper(UWRTCANWrapper&& to_move);
+  explicit UWRTCANWrapper(std::string name, std::string interface_name, bool rcv_big_endian,
+                          int thread_sleep_millis = 10);  // NOLINT(cppcoreguidelines-avoid-magic-numbers,
+                                                          // readability-magic-numbers)
+  explicit UWRTCANWrapper(const UWRTCANWrapper &to_copy) = delete;
+  // NOLINTNEXTLINE(performance-noexcept-move-constructor,
+  // bugprone-exception-escape)
+  explicit UWRTCANWrapper(UWRTCANWrapper &&to_move);
```

Comment:
your clang tidy errors are likely because of this formatting change. The NOLINT comment cannot be broken up